messages/pt.json

```diff
@@ -2,5 +2,9 @@
   "admin/editor.trustpilot.base-url": "URL Base",
   "admin/editor.trustpilot.template-id": "ID do template",
   "admin/editor.trustpilot.business-unit-id": "ID da unidade de negócio",
-  "admin/editor.trustpilot.widget": "Trustpilot - Widget Genérico"
+  "admin/editor.trustpilot.widget": "Trustpilot - Widget Genérico",
+  "admin/editor.trustpilot.theme": "Theme",
+  "admin/editor.trustpilot.width": "Width",
+  "admin/editor.trustpilot.height": "Height",
+  "admin/editor.trustpilot.stars": "Stars"
```

Comment:
Shouldn't these be translated to Portuguese too?

Reply:
added!